Update in initialize-with-readme.md

VimalaMoger opened the pull request:
Document url:
https://docs.github.com/en/repositories/creating-and-managing-repositories/quickstart-for-repositories

### Why:
Under **Create a repository**  section, as per instruction

If I click + icon, then New repository, 
then, in a New repository UI page:
Under Step **2. Configuration**: There were some missing instruction

<img width="774" height="174" alt="newRepos" src="https://github.com/user-attachments/assets/67ed78b7-75ad-4525-bef0-8520a8006d48" />




### What's being changed (if available, include any code snippets, screenshots, or gifs):

Made changes:
In initialize-with-readme.md
Changed the text **Initialize this repository with a README.** to **Add README.**

### Check off the following:

- [x] A subject matter expert (SME) has reviewed the technical accuracy of the content in this PR. In most cases, the author can be the SME. Open source contributions may require an SME review from GitHub staff.
- [x] The changes in this PR meet [the docs fundamentals that are required for all content](http://docs.github.com/en/contributing/writing-for-github-docs/about-githubs-documentation-fundamentals).
- [x] All CI checks are passing and the changes look good in the review environment.

VimalaMoger committed:
updated quickstart-for-repositories and initialize-with-readme.md

Sharra-writes commented on data/reusables/repositories/initialize-with-readme.md:
```diff
@@ -1 +1 @@
-1. Select **Initialize this repository with a README**.
+1. Select **Add README**.
```
```suggestion
1. Toggle **Add README** to **On**.
```
It's a toggle now, not a selection, so it would be better if the wording reflected that for clarity.

Sharra-writes committed:
Update data/reusables/repositories/initialize-with-readme.md

VimalaMoger committed:
Update quickstart-for-repositories.md

I want to keep the information as it is. (in quickstart-for-repositories.md file)

Sharra-writes commented on content/repositories/creating-and-managing-repositories/quickstart-for-repositories.md:
```diff
@@ -25,14 +25,19 @@ topics:
 1. Optionally, add a description of your repository. For example, "My first repository on {% data variables.product.github %}."
 {% data reusables.repositories.choose-repo-visibility %}
 {% data reusables.repositories.initialize-with-readme %}
+1. Add **.gitignore.**
```
```suggestion
```
This is a quickstart guide. This step and the next are entirely optional and not needed for new users looking to create their first repository quickly.

Sharra-writes committed:
Merge branch 'main' into newFix#
